web/html/src/manager/proxy/container-config-messages.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+
+import { Messages } from "components/messages";
+
+type SuccessType = boolean | undefined;
+type MessagesType = React.ReactNode[];
```

Comment:
I think this is not a good name because we already have a very similar name in use with `MessageType` (in `web/html/src/components/messages.tsx`). From the looks of it it looks like there's some inconsistency here in how these two things are used?